### PR TITLE
Built Rest Endpoints For Company Model

### PR DESCRIPTION
**Pull request by chegerofelix**
#### Description
Build rest endpoints for the company model
#### Completed tasks
endpoints for the model are complete and working
#### Testing
copy the endpoints for the api and test it on postman

### CODE REVIEW

**Comment by kelvinndmo** [comment]
@chegrofelix please follow the pull request format

File: company/models.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+class Company(models.Model):
+    INDUSTRY_CHOICES=[
+        ('entertainment', 'Entertainment'),
+        ('manufacturing', 'Manufactiring'),
+        ('real_estate', 'Real_estate'),
+        ('accommodation services', 'Accommodation services'),
+        ('financial and insurance', 'Financial and Insurance'),
+        ('information and communication', 'Information and Communication'),
+    ]
+    companyName = models.CharField(max_length=60)
```
Comment (kelvinndmo) [inline company/models.py]: In python we don't use camel case, change these and everywhere else to company_name and be consistent with that

File: company/tests.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# Create your tests here.
+
+class CompanyTestClass(TestCase):
+    def setUp(self):
+        self.company = Company(companyName='Coca Cola')
+
+    def test_company_instance(self):
+        self.assertTrue(isinstance(self.company, Company))
+
+    def test_save_hood_method(self):
```
Comment (kelvinndmo) [inline company/tests.py]: This test name does not make sense at all, avoid copy pasting code

**Review by kelvinndmo — CHANGES_REQUESTED** [review]
@chegrofelix kindly work on the feedback i have provided

File: company/models.py (hunk 1 above)
Comment (chegerofelix) [reply company/models.py]: okay i'll work on it

**Comment by devsamuelmwangi** [comment]
## Good job but sadly not currently applicable! We don't need company models. Would be redundant!

**Pull request closed without merging**